hack/update-vendor.sh

```diff
@@ -309,6 +309,8 @@ fi
 kube::log::status "go.mod: adding generated comments"
 add_generated_comments "
 // This is a generated file. Do not edit directly.
+// Ensure you've carefully read
+// https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/vendor.md
```

Comment:
```suggestion
// https://git.k8s.io/community/contributors/devel/sig-architecture/vendor.md
```

Reply:
Thanks for pointing out. Didn't know the domain name before. Updated.